design/media_types.go

```diff
@@ -31,3 +31,66 @@ var AuthToken = MediaType("application/vnd.authtoken+json", func() {
 		Attribute("token")
 	})
 })
+
+var WorkItem = MediaType("application/vnd.workitem+json", func() {
+	TypeName("WorkItem")
+	Description("ALM Work Item")
+	Attribute("id", String, "unique id per installation")
```

Comment:
What is the benefit of having a unique ID per installation? This could be problematic if you want to copy over work items from one installation to another. Is this something like the JIRA ID composed of `PROJECT_SHORT_NAME-NUMBER`?